src/runtime/generateTPAParams.ts

```diff
@@ -1,8 +1,9 @@
+import {createStyleParamGetter, StyleParamType, IStyles as IStylesStorage} from '@wix/tpa-settings';
```

Comment:
you checked it also in project that is not using the flow?

Reply:
These projects won't pass `defaults` to `getProcessedCss`.